src/Tree.tsx

```
@@ -579,7 +581,7 @@ class Tree<TreeDataType extends DataNode | BasicDataNode = DataNode> extends Rea
           newExpandedKeys = arrAdd(expandedKeys, node.props.eventKey);
         }
 
-        if (!('expandedKeys' in this.props)) {
+        if (!this.props.hasOwnProperty('expandedKeys')) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)